homeassistant/components/zha/binary_sensor.py

```diff
@@ -49,7 +49,7 @@
 
 async def async_setup_entry(hass, config_entry, async_add_entities):
     """Set up the Zigbee Home Automation binary sensor from config entry."""
-    entities_to_create = hass.data[DATA_ZHA][DOMAIN] = []
+    entities_to_create = hass.data[DATA_ZHA][DOMAIN]
```

Comment:
should these be `entities_to_create = hass.data[DATA_ZHA].get(DOMAIN, [])` to guard them? or are we certain there will always be something there?

Reply:
pre-populating those in integration setup.